Build failed for spark1.4 and MR1

1.environment:hadoop2.7,spark1.4.0(standalone,single node) maven 3.3.3 python 2.6.6
2.execute <HiBench_Root>/bin/build-all.sh.
3.build failed
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 15.274 s
[INFO] Finished at: 2015-07-08T15:05:59+08:00
[INFO] Final Memory: 17M/170M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project sparkbench: Could not resolve dependencies for project com.intel.sparkbench:sparkbench:jar:4.0-SNAPSHOT: The following artifacts could not be resolved: org.apache.spark:spark-core_2.10:jar:1.4.0-SNAPSHOT, org.apache.spark:spark-yarn_2.10:jar:1.4.0-SNAPSHOT, org.apache.spark:spark-streaming_2.10:jar:1.4.0-SNAPSHOT, org.apache.spark:spark-mllib_2.10:jar:1.4.0-SNAPSHOT, org.apache.spark:spark-hive_2.10:jar:1.4.0-SNAPSHOT, org.apache.spark:spark-graphx_2.10:jar:1.4.0-SNAPSHOT: Could not find artifact org.apache.spark:spark-core_2.10:jar:1.4.0-SNAPSHOT in scala-tools.org (https://oss.sonatype.org/content/groups/scala-tools/) -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
Build failed for spark1.4 and MR1, please check!

![buildfail](https://cloud.githubusercontent.com/assets/7839597/8565672/36dc421e-258a-11e5-9baa-97a8b46c0d2c.PNG)

--- a/src/pom.xml
+++ b/src/pom.xml
@@ -161,7 +161,7 @@
             <id>spark1.4</id>
             <properties>
                 <env>spark1.4</env>
-                <spark.version>1.4.0-SNAPSHOT</spark.version>
+                <spark.version>1.4.0</spark.version>
                 <spark.bin.version>1.4</spark.bin.version>
             </properties>
             <activation>

